src/common/dto/resumes/request/create.resume.request.ts

```
@@ -1,24 +1,24 @@
 import { ApiHideProperty, ApiProperty } from '@nestjs/swagger';
 
-import { Transform } from 'class-transformer';
 import {
-    IsBoolean,
     IsEnum,
```

ESLint

[error] 'IsEnum' is defined but never used.

(@typescript-eslint/no-unused-vars)